docs/data/introduction/licensing/licensing.md

```diff
@@ -213,14 +213,15 @@ For example, if you purchase a one-year license today, you will be able to updat
 Those versions will always be available for use in a deployed application,
 however you'll be required to renew your license if you need to continue development with a version released after twelve months.
 
-#### Out of scope license key
+#### License key plan mismatch
 
-This error indicates that the component you are trying to use is not included in your license.
-This can happen if you try to use `DataGridPremium` with a Pro license.
+This error indicates that the component you are trying to use is not included in the plan of your license key.
+For example, this happen if you try to use `DataGridPremium` with a license key for the Pro plan.
```

Comment:
The usage of DataGridPremium with a pro license is the only scenario I see today
Not sure the "For example" make sense.

Reply:
Ok, sold